cirq-web/example.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
I recommend using the standard Cirq notebook header, with links to the devsite / Colab / github. An example can be seen at the top of [this file](https://github.com/quantumlib/Cirq/blob/master/docs/circuits.ipynb).

Reply:
Done